tests/pipeline/Jenkinsfile.longhaul

```diff
@@ -7,9 +7,10 @@ pipeline {
           choice(name: 'key', choices: 'master\nPR', description: 'Select master or a PR to run tests aganist - defaults to master')
           string(name: 'PR', defaultValue: '', description: 'PR number, required if key value is set to PR')
           string(name: 'version', defaultValue: 'v0.5.0-111.ga4a1a98', description: 'Version to run tests against')
-          string(name: 'Load_test', defaultValue: 'BlockLongHaul', description: 'select a load test to run ')
-          string(name: 'testCount', defaultValue: '50', description: 'Number of times to repeat test per run')
+          string(name: 'Load_test', defaultValue: 'TestBlockLongHaul', description: 'select a load test to run ')
+          string(name: 'testCount', defaultValue: '10', description: 'Number of times to repeat test per run')
```

Comment:
how does the longhaul determine how long to run? A certain number of iterations? I wonder if we could have a time-based longhaul